packages/google-cloud-serverless/src/index.ts

```diff
@@ -47,7 +47,6 @@ export {
   winterCGHeadersToDict,
   // eslint-disable-next-line deprecation/deprecation
   anrIntegration,
-  // eslint-disable-next-line deprecation/deprecation
```

Comment:
Hmm I guess it doesn't flag the deprecation here anymore because only some signatures of this API are deprecated 🤔
I think that's probably fine, was just curious if you think there might be another reason.

 https://github.com/getsentry/sentry-javascript/blob/a1cc67548ff347fb7226a2143babe227dc8651fc/packages/node-core/src/integrations/anr/index.ts#L268-L279

Reply:
Very good point - I'll investigate into that one